deeppavlov/models/faq/faq_cos.py

```diff
@@ -68,11 +99,15 @@ def fit(self, x_train_vects, y_train) -> None:
         self.y_train = list(y_train)
 
     def save(self) -> None:
+        """Save FAQ model
```

Comment:
Looks ugly, isn't it? Try one line... Also, avoid obvious descriptions.

Reply:
done.